components/bottom-nav.tsx

```
@@ -0,0 +1,101 @@
+              "inline-flex flex-col items-center justify-center px-2 hover:bg-muted",
+              route.active ? "text-primary" : "text-muted-foreground"
+            )}
+          >
+            {route.icon}
+            <span className="text-xs">{route.label}</span>
+          </Link>
+        ))}
+        <button
+          onClick={postAdRoute.onClick}
+          className="inline-flex flex-col items-center justify-center px-2 hover:bg-muted text-muted-foreground"
+        >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)